src/Orleans/GrainDirectory/MultiClusterStatus.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Orleans.GrainDirectory
+{
+    /// <summary>
+    /// Status of a directory entry with respect to multi-cluster registration
+    /// </summary>
+    internal enum MultiClusterStatus : byte
+    {
+        /// <summary>
+        /// Registration is owned by this cluster.
+        /// </summary>
+        Owned,
+
+        /// <summary>
+        /// Failed to contact one or more clusters while registering, so may be a duplicate.
+        /// </summary>
+        Doubtful,
```

Comment:
These names are chosen mainly for consistency with the protocol description as it appeared in the paper (going back to Jose's internship). We can easily change them. Doubtful was chosen because it is the result of an unconclusive communication attempt.  Unknown seems fine as well.